lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +45,33 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def enough_approvals?
```

Comment:
@codenamev Good catch with the test! I am adding tests to cover: `check_approvals`